python-avd/pyavd/_eos_designs/structured_config/network_services/ethernet_interfaces.py

```diff
@@ -134,10 +134,13 @@ def ethernet_interfaces(self: AvdStructuredConfigNetworkServices) -> list | None
 
                             if get(l3_interface, "pim.enabled"):
                                 if not vrf.get("_evpn_l3_multicast_enabled"):
-                                    msg = (
-                                        f"'pim: enabled' set on l3_interface '{interface_name}' on '{self.shared_utils.hostname}' requires evpn_l3_multicast:"
-                                        f" enabled: true under VRF '{vrf['name']}' or Tenant '{tenant['name']}'"
-                                    )
+                                    if not self.shared_utils.evpn_multicast:
+                                        msg = f"'pim: enabled' set on l3_interface '{interface_name}' on '{self.shared_utils.hostname}' requires evpn_multicast: true"
```

Comment:
Should this message be `requires evpn_multicast: true at the fabric level` ?

Reply:
added at the fabric level